### DISCUSSION TITLE
Slack

### DISCUSSION THREAD

**Discussion by Jkarlsson1** [atralice/Curso.Prep.Henry#101] [Q&A]
Hola, buenas tardes. Soy nuevo en la plataforma y estoy haciendo las homework pero tengo varias dudas y quería saber como entrar con la plataforma de slack pq nunca me llego el mail con la invitación y ya me fije en la casilla de spam y sigo sin encontrarla. No se si alguien me podría ayudar

**Comment by Tayra27** [comment]
Yo estoy en la misma

**Reply by aliado28** [reply]
pudiste encontrar la forma? tengo el mismo problema

**Reply by SelqofJ** [reply]
en el mail de precourse abajo esta el link de slack

**Reply by ghost** [reply]
holaa, cuando estoy en admisión en heny no puedo ingresar al slack, como hago?

**Comment by LoOW3** [comment]
En el último mail que te llega, que tiene este enunciado "Realiza el último paso de tu aplicación a Henry" si scrolleas hay un botón amarillo que dice "acceder a slack"

**Reply by Banco-Nacion-HomeBanking** [reply]
Mil gracias Genio..

**Reply by lorzap134** [reply]
Gracias

**Comment by jorgebarrios94** [comment]
buen día como puedo unirme al canal del programa por slack

**Comment by mauricioalejandrodiaz** [comment]
A mi me mandaron el correo con el enlace y no esta el link. No se como agregar slack. Alguien sabe algún método alternativo para entrar al slack?

**Comment by julian0986** [comment]
Hola a todos, soy nuevo pero no me deja ingresar a Slack por el link que enviaron al correo le doy continuar con google pero me sale un error.
![error al ingresar](https://user-images.githubusercontent.com/79938910/221680955-97a964f8-0275-443f-9a3d-9f33261f6b7e.png)

**Reply by HelloNerk** [reply]
estoy en las mismas 😢

**Comment by julian0986** [comment]
Ya escribí a Henry a ver si nos ayudan

El lun, 27 feb 2023 a la(s) 20:03, HelloNerk ***@***.***)
escribió:

> estoy en las mismas 😢
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/atralice/Curso.Prep.Henry/discussions/101#discussioncomment-5135892>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATB4KXWDSU2I5PRKHV5BHDLWZVFG5ANCNFSM5BKOLIQA>
> .
> You are receiving this because you commented.Message ID: <atralice/Curso.
> ***@***.***>
>


-- 
Jose Julian Calle Mesa
Móvil: 311 606 06 42

**Reply by emavaccarini** [reply]
hola una consulta te sirvio escribirles en ese tiempo? como pudiste entrar despues?
